### DISCUSSION TITLE
Ruffle in Launchbox - Need Help Plz

### DISCUSSION THREAD

**Discussion by EvoRetroGame** [ruffle-rs/ruffle#15822] [Q&A] [answered]
Hi, I try to get run Ruffle in fullscreen with scale=showAll and forceScale=true.
Here is my default line parameters command in launchbox:  --fullscreen --scale=showAll --forceScale=true
Like this Ruffle won't launch at all, if remove command line than emu is running without any issue but no parameters apply....
Really need help on this one please.

**Comment by n0samu** [comment, marked as answer]
The correct arguments to use are:
`--fullscreen --scale=show-all --force-scale`

**Reply by EvoRetroGame** [reply]
Thanks so much , it work;)
Cheers

**Comment by sersoft2008** [comment]
i have used The correct arguments to use are:
--fullscreen --scale=show-all --force-scale but the games swf don't start automatically
